### ISSUE TITLE
Implement units in other way

### ISSUE THREAD

**Issue by Fratorhe** [SamTov/MDSuite#15]
Sam, 

I would suggest to implement the different units systems of LAMMPS as input to the code. This is, for the Class constructor to pass an argument of unit_type = "metal", "real", etc. And this, internally can create the self.time_unit, self.length_unit, etc. accordingly. If required, we can leave the option "other" so the user can introduce the units in a dictionary and pass it as **kwargs, or pass it as a file (unit_type="file_unit"), and it can be read from a json file in the same directory.

Benefits: 
- More standardized class
- Less inputs
- Less prone to unit errors
- Should not be a big change to the code. 

Drawbacks:
- Focused on LAMMPS. 
- Some code re-writing.

If you like the idea, I can take care of the implementation

**Comment by SamTov** [comment]
That’s definitely a good idea, I wouldn’t think it too much work. I am okay with a LAMMPS focus as when added formats exist we will adapt it to suit.

**Comment by Fratorhe** [comment]
Ok, I will implement it in the coming days. 
I plan to implement it as a "dispatch dictionary" or something similar, so that it can be updated easily in the future if new unit_types are created. 
Is it ok for you?

**Comment by SamTov** [comment]
Yeah that is good for me, switch/case styled programming is always nicer than elif's
